utils.h

```diff
@@ -28,13 +28,14 @@
 #include <stdlib.h>
 #include <string.h>
 #include <unistd.h>
+#include <syslog.h>
 #include <sys/types.h>
 #include <sys/stat.h>
 
 #if 0
-#define __debug__(x) printf x
+#define __debug__(...) bwrap_log (LOG_DEBUG, __VA_ARGS__)
```

Comment:
This needs to have the calls to `__debug__(...)` drop their inner parens. Right now they look like:

```c
__debug__ (("launch executable %s\n", argv[0]));
```

which would now end up expanding to use of the comma operator

```c
bwrap_log (LOG_DEBUG, ("launch executable %s\n", argv[0]));
```

Reply:
Good point. I'm very tempted to take this opportunity to rename them to `DEBUG()` or `trace()`, or anything else that isn't in the reserved-for-the-toolchain namespace.